cime_config/machines/cmake_macros/intel_chrysalis.cmake

```diff
@@ -0,0 +1,35 @@
+if (COMP_NAME STREQUAL gptl)
+  string(APPEND CPPDEFS " -DHAVE_SLASHPROC")
+endif()
+string(APPEND CFLAGS " -static-intel")
+string(APPEND CFLAGS " -march=core-avx2")
```

Comment:
Can lines 4 and 5 be combined in to one?

Reply:
yes, it should be safe to group sequential appends with the same selectors (or lack of selectors in this case).